platform/broadcom/sonic-platform-modules-accton/debian/sonic-platform-accton-as7326-56x.postinst

```diff
@@ -2,27 +2,7 @@
 # Disable monitor, monitor-fan, monitor-psu (not enabling them would imply they will be disabled by default)
 # Enable pddf-platform-monitor
 
-# Steps to check syseeprom i2c address
-modprobe i2c-i801
-modprobe i2c-dev
-use_57_eeprom=true
-(i2cget -y -f 0 0x56 0x0) > /dev/null 2>&1
-if [ $? -eq 0 ]; then
-    use_57_eeprom=false
-fi
-
-if $use_57_eeprom ; then
-    echo "The board has system EEPROM at I2C address 0x57"
-    # syseeprom is at the i2c address 0x57. Change the PDDF JSON file
-    sed -i 's@"topo_info": {"parent_bus": "0x0", "dev_addr": "0x56", "dev_type": "24c04"},@\
-        "topo_info": {"parent_bus": "0x0", "dev_addr": "0x57", "dev_type": "24c02"},@g' \
-        /usr/share/sonic/device/x86_64-accton_as7326_56x-r0/pddf/pddf-device.json
-    sync
-fi
-
 depmod -a
-systemctl enable as7326-platform-handle_mac.service
-systemctl start as7326-platform-handle_mac.service
```

Comment:
Why this is removed? Please include it.

Reply:
Add handle_mac  back to the .posinst file.